Inventory/InMemoryInventory.cs

```diff
@@ -46,12 +48,8 @@ public void PrintInventory()
         }
     }
 
-    public void EditProduct(Product product, String? name, String? description, double price)
+    public void EditProduct(Product product, String? description, double price)
```

Comment:
why have you removed the ability to edit the name? this is a requirement!!

Reply:
this is because I used the name as a primary key instead of having an ID field. I will fix it